framework/include/materials/MaterialData.h

```diff
@@ -42,53 +42,56 @@ class MaterialData
   void release();
 
   /**
-   * Size the properties
+   * Resize the data to hold properties for n_qpoints quadrature points.
    */
   void size(unsigned int n_qpoints);
```

Comment:
I'd really like to rename this to `resize()`.  I know you didn't name it... but it's a good time to do it.